getting_started/step_by_step/resources.rst

```diff
@@ -82,21 +82,36 @@ Loading resources from code
 Loading resources from code is easy. There are two ways to do it. The
 first is to use load(), like this:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     func _ready():
             var res = load("res://robi.png") # resource is loaded when line is executed
             get_node("sprite").texture = res
 
+ .. code-tab:: csharp
+
+    public override void _Ready()
+    {
+        var texture = (Texture) GD.Load("res://robi.png"); // resource is loaded when line is executed
```

Comment:
Classic nitpicking: there should be no space between `(Texture)` and `GD`.  
Same on the next line.